src/insights_client/__init__.py

```diff
@@ -351,37 +349,62 @@ def _main():
     attempt to collect and upload with new, then current, then rpm
     if an egg fails a phase never try it again
     """
+    set_up_logging()
+    logger.debug("Verifying initial eggs to load config.")
+
+    validated_eggs = []
+    for egg in [STABLE_EGG, RPM_EGG]:
+        logger.debug("GPG verifying initial egg %s..." % (egg,))
+        if gpg_validate(egg):
+            logger.debug("GPG verification successful. Loading initial egg: %s" % (egg,))
+            validated_eggs.append(egg)
+        else:
+            logger.debug("WARNING: GPG verification failed. Not loading initial egg: %s" % (egg,))
```

Comment:
The `gpg_validate` already logs if the gpg passed successfully or not.

Reply:
Right. The information is redundant. Only the latter part of both log messages brings new information.